apps/web/src/hooks/useDocumentParameters/index.ts

```diff
@@ -1,59 +1,10 @@
 import { useCallback, useMemo } from 'react'
 
 import { recalculateInputs } from '$/hooks/useDocumentParameters/recalculateInputs'
-import { DocumentLog, ParameterType } from '@latitude-data/core/browser'
+import { DatasetSource, DocumentLog, INPUT_SOURCE, Inputs, InputSource, PlaygroundInput, PlaygroundInputs } from '@latitude-data/core/browser'
```

Comment:
I moved all the types and constants related to document parameters to the core. 
The reason is that now I want to type the jsonb field in the DB with those types. So, having it on web was not possible.